From 5d48d1fd5ca761faffc23fd54304cc3c37a01784 Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Fri, 3 Jun 2016 11:18:02 +0800
Subject: [PATCH 1081/5242] MLK-12880 arm: dts: imx7d: correct the
 PAD_GPIO1_IO01 pin ctrl setting

commit  029bc6e1f685e5c217b34084cc8cafc74d1fc7c7 from
https://source.codeaurora.org/external/imx/linux-imx.git

PAD_GPIO1_IO01 bit[31:7] are reserved, remove the setting mapping to
this reserved field.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    2 +-
 arch/arm/boot/dts/imx7d-sdb.dts               |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 0e2c7cb..dd0e148 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -840,7 +840,7 @@
 	imx7d-12x12-lpddr3-arm2 {
 		pinctrl_pwm1: pwm1grp {
 			fsl,pins = <
-				MX7D_PAD_LPSR_GPIO1_IO01__PWM1_OUT	0x110b0
+				MX7D_PAD_LPSR_GPIO1_IO01__PWM1_OUT	0x30
 			>;
 		};
 	};
diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index f3da7bf..86a8bf7 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -634,7 +634,7 @@
 
 		pinctrl_pwm1: pwm1grp {
 			fsl,pins = <
-				MX7D_PAD_LPSR_GPIO1_IO01__PWM1_OUT	  0x110b0
+				MX7D_PAD_LPSR_GPIO1_IO01__PWM1_OUT	  0x30
 			>;
 		};
 
-- 
1.7.9.5

